core/src/main/java/nl/nn/adapterframework/jdbc/JdbcTableMessageBrowser.java

```diff
@@ -88,13 +92,19 @@ protected void setOperationControls() {
 		useFirstRowsHint = ac.getBoolean(PROPERTY_USE_FIRST_ROWS_HINT, true);
 	}
 
-
+	@Override
+	protected M retrieveObject(ResultSet rs, int columnIndex) throws JdbcException, SQLException {
+		if (tableListener!=null) {
+			return tableListener.extractRawMessage(rs);
+		}
+		return (M)rs.getString(columnIndex);
```

Comment:
kan je een string zomaar casten naar een M ?

Reply:
nee, maar tableListener is ook nooit null